StackCalculator/StackCalculator.Tests/StackCalculatorTest.cs

```diff
@@ -0,0 +1,76 @@
+using Calculator.exceptions;
+
+namespace Calculator.Tests;
+
+public class StackCalculatorTest
+{
+    private const double DELTA = 0.001;
+    
+    private bool ApproximatelyEqual(double first, double second)
+    {
+        return first >= second - DELTA && first <= second + DELTA;
+    }
```

Comment:
```suggestion
    private bool ApproximatelyEqual(double first, double second)
        => first >= second - DELTA && first <= second + DELTA;
```
Используйте expression-bodied methods везде, где это можно (то есть метод из одного оператора)